test/actions/parse_csv_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule WorkflowEngine.Actions.ParseCsvTest do
```

Comment:
In my opinion, we should write more tests for this. The 3 tests that are implemented are all success cases.
From painful Maintain-Tool experience, we could use additional error-case tests (maybe all of this is handled with NimbleCSV, it wasn't with the CSV library):
- Empty file
- Different encoding (latin and UTF), missing or existing BOM character
- Empty cells and missing values
- Missing headers and/or rows with more/less columns than required
- Quotation marks and how they're parsed (do they need to be doubled?), different escape characters
- etc